sigal/writer.py

```diff
@@ -81,6 +83,9 @@ def __init__(self, settings, index_title=''):
             **env_options
         )
 
+        for filter in filters:
+            env.filters[filter.__name__] = filter
```

Comment:
Having the possibility to set Jinja2 filters could be useful, but it should be possible to do it with a setting. So I would prefer that we have a list of builtin filters *not* accessible from a global var (remove `sigal.filters.filters`), and allow to add more filters with a setting.